src/migrations/20201123125725-create-reviews.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
           key:'id'
         }
       },
-      AccommodationId: {
+      accommodationId: {
```

Comment:
@CYIMANA-Faisal create a new migration file that changes the name of this column